src/robot_arm.cpp

```diff
@@ -1,37 +1,79 @@
 /**
  * @file
  * @brief     Robot Arm class
- * @author    Jeroen van Hattem and Jeffrey de Waal
- * @license   MIT
+ * @author    Jeroen van Hattem, Jeffrey de Waal, Wiebe van Breukelen
+ * @license   See LICENSE
  */
 #include "robot_arm.hpp"
 
 namespace RobotArm {
 
-RobotArm::RobotArm() : conn(115200, UARTController::ONE) {
+RobotArm::RobotArm() : uartConn(115200, UARTController::ONE) {
 }
 
 void RobotArm::sendGCodeToArm(const char *command) {
-    conn << command;
+    uartConn << command;
 }
 
-void RobotArm::move(const Coordinate3D coordinates, int speed) {
-    speed = speed;
+void RobotArm::move(Coordinate3D coordinates, unsigned int speed) {
+    this->speed = speed;
     determineGCode(coordinates, speed);
 
     sendGCodeToArm(commandBuffer);
 }
 
-void RobotArm::executeAction(const char *action) {
+Coordinate3D RobotArm::getPosition() {
+    char response[40];
+    uartConn << "#n P2220\n"; /// See Goode commands on page 26 developer guide -
+                              /// http://download.ufactory.cc/docs/en/uArm-Swift-Pro-Develper-Guide-171013.pdf
+
+    receiveGcodeResponse(response, 40);
+
+    Coordinate3D coordinate;
+    int posXStart, posYStart, posZStart;
+    int posXEnd, posYEnd, posZEnd;
+
+    /// Example response: $n ok X100.00 Y100.00 Z100.00\n
```

Comment:
According to the code standard, inline comments are formatted like this:
///< Comment

Reply:
Good point, will be addressed.